src/index.html

```diff
@@ -7,7 +7,119 @@
     <title>Stars</title>
     <link rel="stylesheet" href="./style.css">
   </head>
-  <body>
-    <h1>Stars</h1>
+  <body class="body">
+
+    <div class="stars__blocks">
+      <div class="stars stars--0 ">
+        <div class="stars__star ">
+
+        </div>
+        <div class="stars__star">
+
+        </div>
+        <div class="stars__star">
+
+        </div>
+        <div class="stars__star">
+
+        </div>
+        <div class="stars__star ">
+
+        </div>
+
+      </div>
```

Comment:
```suggestion
      <div class="stars stars--0 ">
        <div class="stars__star "></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star"></div>
        <div class="stars__star "></div>
      </div>
```
Code readability can be improved here